File: python/contaBancaria.py
import itertools 

class ContaBancaria:
    id_generator = itertools.count(10000)
    def __init__(self, agencia, cpf, nome, saldo):
        self.agencia = agencia
        self.numero = next(self.id_generator)
        self.cpf = cpf
        self.nome = nome
        self.saldo = saldo

    def saque(self, valor):
        if((self.saldo-valor)>0):
            self.saldo -=valor
            print('Saque realizado com sucesso!\nSeu saldo é de:', self.saldo)
            return True
        else:
            print("Saldo insuficiente.\nSeu saldo atual é de " + self.saldo)
            return False

    def deposito(self, valor):
        self.saldo +=valor
        print('deposito realizado com sucesso!\nSeu saldo é de:', self.saldo)
    def imprime(self):
        print('Numero da Agencia', self.agencia)
        print('Numero da conta: ', self.numero)
        print('CPF: ', self.cpf)
        print('Nome: ', self.nome)
        print('Saldo = ', self.saldo)

conta = ContaBancaria(1001,13,'jose',10)
conta.deposito(10)
conta.imprime()

File: c/contaBancaria.c
#include <stdio.h>
#include <stdlib.h>
#include <cstdlib>
#include <iostream>
#include "Contabancaria.h"
using namespace std;
void Inicializa(ContaBancaria* conta, int cpf, double saldo)
{
    (*conta).cpf = cpf;
    (*conta).saldo = saldo;
}
void Deposito (ContaBancaria* conta, double valor)
{
    (*conta).saldo += valor;
}
int Saque (ContaBancaria* conta, double valor)
{
    double saldo;
    saldo = (*conta).saldo;
    if ((saldo - valor) <0)
    {
        printf("Saldo Insuficiente");
        return 0; 
    } else
    {
    (*conta).saldo -= valor;
        return 1; 
    }
}
void Imprime (ContaBancaria conta)
{
    printf("CPF: %d\n", conta.cpf);
    printf("Saldo: %.2f\n", conta.saldo);
}
void Transferir (ContaBancaria* envia, double valor, ContaBancaria* recebe)
{
    if (Saque(envia,valor))
    {
        Deposito(recebe, valor);
    }
}

void initlista(lista *pRecebido)
{
    (pRecebido)->proximo = NULL;
}
void insertlista(lista *pRecebido)
{
//Declaracoes
    lista *temporario;
    ContaBancaria valor;
//Instrucoes
    cout<<"\n\tInforme um valor a ser inserido : ";
    cin>>valor;
    temporario = (lista * )malloc(sizeof(lista));
    temporario->conta = valor;
    temporario->proximo = (pRecebido)->proximo;
    (pRecebido)->proximo = temporario;
}
void buscalistasimples(lista *pRecebido)
{
    //Declaracoes
    lista *temporario;
    int i=0;
    //Instrucoes
    if((pRecebido)->proximo == NULL)
    {
        cout<<"\tLista Vazia!\n";
    }else
    {
        temporario = (lista * )malloc(sizeof(lista));
        temporario = (pRecebido)->proximo;
        system("cls");
        while(temporario!=NULL)
        {
            cout<<"\n\t["<<(i=i+1) <<"] Valor : "<<temporario->conta<<"\n";
            temporario = temporario->proximo;
        }
    }
}